lib/rubocop/cop/mixin/target_ruby_version.rb

```diff
@@ -5,15 +5,24 @@ module Cop
     # Common functionality for checking target ruby version.
     module TargetRubyVersion
       def required_minimum_ruby_version
-        @minimum_target_ruby_version
+        @minimum_target_ruby_version || 0
```

Comment:
The value `0` for the name `minimum_target_ruby_version` seems inappropriate. How about setting it to `2.0`, the minimum Ruby version supported for analysis?

```suggestion
        @minimum_target_ruby_version || 2.0
```

Reply:
@koic @bbatsov I used 0 here because the idea is to just have a fixed minimum to compare against. I can do 2.0 (I understand the logic) but then the change to [cops_documentation_generator.rb](https://github.com/rubocop/rubocop/pull/13473/files#diff-664aaca7a3829e6afc7275b3aebc2ee646de27f55ca01361d7fff64e0f7f8641) needs to hardcode 2.0 as a minimum, which means there's an edge case if someone actually was to set `minimum_target_ruby_version 2.0` (which `Style/Dir` and `Style/SymbolArray`) do. Also, if we're going to show the minimum as 2.0 instead, it'd only show up in the documentation when a cop class is extended with `TargetRubyVersion`, which is a minority of cops, so the documentation would be inconsistent.

This would only affect cops that have a maximum version set but no minimum (which at the moment is only this one).

So IMO we need a value that allows comparison but will not affect how the documentation is rendered. If 0 doesn't work for you, how about `-Infinity`?

Reply:
I'm OK with 0 as well - just add comment explaining this.